pkg/controllers/managementuser/nodesyncer/cordonfieldssyncer.go

```diff
@@ -275,24 +275,15 @@ func filterErrorMsg(msg string, nodeName string) string {
 	return strings.Join(upd, "\n")
 }
 
-func removeDrainCondition(obj *v3.Node) {
-	exists := false
-	for _, condition := range obj.Status.Conditions {
+func removeDrainCondition(node *v32.Node) {
```

Comment:
Looks like both v3 and v32 point to the same type in the end: https://github.com/rancher/rancher/blob/d30a8488554ba2fe8304d978c273687616cc95d4/pkg/apis/management.cattle.io/v3/machine_types.go#L62  

This is because we have https://github.com/rancher/rancher/blob/d30a8488554ba2fe8304d978c273687616cc95d4/pkg/generated/norman/management.cattle.io/v3/zz_generated_node_controller.go#L47 

I see that we've already updated nodessyncer.go to use the `v32` or `apimgmtv3` node, so this change is okay. If possible, we should also update other places in the file.

Reply:
I updated the v3 to v32 at all the locations in cordonfieldssyncer.go.